rust_snuba/rust_arroyo/src/processing/strategies/healthcheck.rs

```diff
@@ -0,0 +1,75 @@
+use std::path::PathBuf;
+use std::time::{Duration, SystemTime};
+
+use crate::processing::strategies::{
+    CommitRequest, InvalidMessage, ProcessingStrategy, SubmitError,
+};
+use crate::types::Message;
+use crate::utils::metrics::{get_metrics, BoxMetrics};
+
+const TOUCH_INTERVAL: Duration = Duration::from_secs(1);
+
+pub struct HealthCheck<Next> {
+    next_step: Next,
+    path: PathBuf,
+    interval: Duration,
```

Comment:
Is this field necessary? It's never anything but `TOUCH_INTERVAL`.

Reply:
maybe we want to make this configurable at some point, but you are right, right now its constant.